src/cargo/core/workspace.rs

```diff
@@ -307,9 +307,10 @@ impl<'gctx> Workspace<'gctx> {
                 }
             }
         }
-        if let CargoResolverConfig {
-            incompatible_rust_versions: Some(incompatible_rust_versions),
-        } = self.gctx().get::<CargoResolverConfig>("resolver")?
+        if let Some(incompatible_rust_versions) = self
```

Comment:
Seems unrelated to the feature unification behavior. Is this just a cosmetic change? If so, I'd prefer not doing this, or having it in a separate PR with a reason for the change.

Reply:
It was originally a cosmetic change and I reverted it, but I ended up having to change this area again when implementing the unstable feature gate.